app/participants/Knight.py

```diff
@@ -0,0 +1,37 @@
+class Knight:
+
+    def __init__(self, name: str, power: int, hp: int,
+                 armour: list, weapon: dict,
+                 potion: (dict, None) = None) -> None:
+
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion
+        self.protection = 0
+
+    def apply_armour(self) -> None:
+
+        for a in self.armour:
```

Comment:
did you read checklist?

Reply:
I'm sorry I didn't notice that.
Fixed it, thanks)